package/Dockerfile

```
@@ -27,7 +27,7 @@ ENV CATTLE_CHART_DEFAULT_BRANCH=$CHART_DEFAULT_BRANCH
 ENV CATTLE_PARTNER_CHART_DEFAULT_BRANCH=$PARTNER_CHART_DEFAULT_BRANCH
 ENV CATTLE_RKE2_CHART_DEFAULT_BRANCH=$RKE2_CHART_DEFAULT_BRANCH
 ENV CATTLE_HELM_VERSION v2.16.8-rancher1
-ENV CATTLE_K3S_VERSION v1.21.7+k3s1
+ENV CATTLE_K3S_VERSION v1.22.4+k3s1
 ENV CATTLE_MACHINE_VERSION v0.15.0-rancher73
 ENV CATTLE_MACHINE_PROVISION_IMAGE rancher/machine:${CATTLE_MACHINE_VERSION}
 ENV CATTLE_ETCD_VERSION v3.4.13
```

K3s 1.22 actually embeds etcd 3.5, so we should probably package a similar version of etcdctl
```suggestion
ENV CATTLE_ETCD_VERSION v3.5.1
```